odoo/addons/base/tests/test_views.py

```diff
@@ -4589,8 +4590,12 @@ def test_uncommented_invisible_field(self):
             _logger.info('%s\n%s', msg, msg_info)
 
         if module_error_views:
-            msg_info = '\n'.join(f'Addons: {module!r}   Views: {names}' for module, names in module_error_views.items())
-            _logger.error('%s\n%s', msg, msg_info)
+            error_lines = []
+            for module, view_errors in module_error_views.items():
+                error_lines.append(f"Addon: {module!r}")
+                for view, fields in view_errors.items():
+                    error_lines.append(f"   View: {view}\n      Fields:\n" + "\n".join(f"         {field}" for field in fields))
```

Comment:
Nitpicking: Use `error_lines.extend` and `' ' * 4 + 'View...'` which is more readable than counting spaces.